app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt

```diff
@@ -137,12 +137,12 @@ fun HomeDrawer(
                 val dir = File(logFilePath).parentFile
 
                 if (dir != null) {
-                val logsUris = context.getUrisOfFilesInDirectory(dir)
-                val intent = context.multipleFileSharingIntent(logsUris)
-                intent.putExtra(Intent.EXTRA_EMAIL, arrayOf("wire-newandroid@wearezeta.zendesk.com"))
-                intent.putExtra(Intent.EXTRA_SUBJECT, "Bug Report - Wire Beta")
-                intent.putExtra(Intent.EXTRA_TEXT, EmailComposer.reportBugEmailTemplate(context.getDeviceId()?.sha256()))
-                intent.type = "message/rfc822"
+                    val logsUris = context.getUrisOfFilesInDirectory(dir)
+                    val intent = context.multipleFileSharingIntent(logsUris)
+                    intent.putExtra(Intent.EXTRA_EMAIL, arrayOf("wire-newandroid@wearezeta.zendesk.com"))
+                    intent.putExtra(Intent.EXTRA_SUBJECT, "Bug Report - Wire Beta")
```

Comment:
Do we want to localise this for also German? or is it rather universal tech language 😅 ?